analysis/all_time_counts_v2.py

```diff
@@ -0,0 +1,145 @@
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from common_variables import demographic_variables, clinical_variables
+
+pd.set_option("display.max_rows", 50)
+results_path = "output/practice_summ_v2.txt"
```

Comment:
This file is almost identical to the original V1 version, except for updating inputs/output filenames to refer to v2 versions, and the few lines commented below